getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,83 @@
+
+The VPP dataplane is entirely compatible with the other Calico dataplanes, meaning that it is possible to have a cluster with some VPP-enabled nodes and some regular nodes. This makes it easy to migrate a cluster from Linux or eBPF Calico networking to VPP networking.
+
+Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the Calico/VPP team.
+
+In the future, the VPP dataplane will offer additional features for network-intensive applications, such as providing `memif` userspace packet interfaces to the pods (instead of regular Linux network devices).
+
+
+### Before you begin...
+
+In the tech preview release, VPP mode has the following pre-requisites:
+
+- Must use the Calico CNI plugin and Calico IPAM.  It is not yet compatible with third-party CNI plugins (AWS CNI/Azure CNI/GKE CNI/flannel etc).
+- The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **VPP dataplane**
+
+### Concepts
+
+#### VPP
+
+The Vector Packet Processor (VPP) is a high-performance, open-source  userspace network dataplane written in C and developped under the [fd.io](https://fd.io) umbrella. It supports many standard networking features (L2, L3 routing, NAT, encapsulations), and it is also easily extensible through a plugin system. To use it as a dataplane for {{site.prodname}}, we developped a couple plugins to efficiently implement K8s services load balancing and {{site.prodname}} policies.
+
+### Installing Calico-VPP on a kubernetes cluster
+
+If you want to try Calico-VPP and see how it works, the simplest way to do so is to use the [VM-based test setup]({{ site.baseurl }}/getting-started/kubernetes/vpp/vm). This requires an Ubuntu 18.04 or 20.04 host, and will create a small 3-VM cluster with virtio interfaces that you can play with.
+
+If you are using hosted kubernetes clusters in public clouds, we provide ready-to go configurations for the major cloud platforms:
+* [Amazon EKS]({{ site.baseurl }}/getting-started/kubernetes/vpp/eks)
+* Google GKE: coming soon!
+* Azure AKS: coming soon!
+
+If the above doesn't apply and you want to test on a specific cluster, then you can use the generic configuration for a baremetal cluster: [Baremetal Cluster setup]({{ site.baseurl }}/getting-started/kubernetes/vpp/baremetal)
+
+If you have a pre-existing Calico cluster, it is also possible to [migrate it to the VPP dataplane]({{ site.baseurl }}/reference/vpp/migrating-cluster).
+
+
+### Next steps
+
+**Testing**
+
+Once Calico-VPP is up and running, you should be able to create pods and services on this kubernetes cluster, just as you would on any cluster. VPP will handle all the container traffic. You can check that pods can reach each other, and that ServiceIPs are properly balanced across backend pods.
```

Comment:
After Calico-VPP is up and running, you can create pods and services on this Kubernetes cluster, just as you would on any cluster. VPP will handle all the container traffic. You can check that pods can reach each other, and that ServiceIPs are properly balanced across backend pods.